cl/corpus_importer/management/commands/update_opinions_order.py

```diff
@@ -0,0 +1,96 @@
+from django.db.models import Count, Q
+
+from cl.lib.command_utils import VerboseCommand, logger
+from cl.search.models import SOURCES, Opinion, OpinionCluster
+
+
+def sort_harvard_opinions(options) -> None:
+    """Sort harvard opinions
+
+    We assume that harvard data is already ordered, we just need to fill
+    the order field in each opinion
+
+    The harvard importer created the opinions in order of appearance in the file
+
+    :param options: dict of arguments skip until and limit if given
+    :return: None
+    """
+
+    skip_until = options.get("skip_until", None)
+    limit = options.get("limit", None)
+
+    base_filter = (
+        OpinionCluster.objects.exclude(filepath_json_harvard="")
+        .annotate(opinions_count=Count("sub_opinions"))
+        .filter(opinions_count__gt=1)
+    )
+
+    if skip_until:
+        base_filter &= Q(pk__gte=skip_until)
```

Comment:
Not that it matters, with the refactor I suggest below, but I don't think this would have worked? Can you add a Q to a QuerySet? Hmmm....